x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/translations.ts

```diff
@@ -89,6 +89,7 @@ export const ADD_RULE_EXCEPTION_FROM_ALERT_COMMENT = (alertId: string) =>
     'xpack.securitySolution.ruleExceptions.addExceptionFlyout.addRuleExceptionFromAlertComment',
     {
       values: { alertId },
-      defaultMessage: 'Exception conditions are pre-filled with relevant data from {alertId}.',
+      defaultMessage:
+        'Exception conditions are pre-filled with relevant data from alert with "id" {alertId}.',
```

Comment:
@WafaaNasr @yctercero what's the field that stores the alert ID value? Is it the `_id` field? If it is, here's the revision I suggest:

        'Exception conditions are pre-filled with relevant data from an alert with the alert id (_id): {alertId}.',
        
        
If you think including the field name is too much or would be confusing, you can remove it, for ex:

        'Exception conditions are pre-filled with relevant data from an alert with the alert id: {alertId}.',

Reply:
One more note: In the first option, I included the field that stores the alert value in case users need to know what field value pair to query when searching for the alert.